sdk/src/main/java/com/mapbox/maps/renderer/egl/EGLConfigChooser.kt

```diff
@@ -31,14 +32,26 @@ internal class EGLConfigChooser constructor(
         EGL_BLUE_SIZE, 5,
         EGL_ALPHA_SIZE, if (translucentSurface) 8 else 0,
         EGL_DEPTH_SIZE, 16,
-        if (antialiasingEnabled) { EGL_SAMPLE_BUFFERS } else { EGL_NONE }, 1,
```

Comment:
Issue is that `EGL_NONE` here prevent some drivers from reading the full attribute list further.